src/rotator_library/providers/anthropic_provider.py

```
@@ -0,0 +1,948 @@
+# Copyright (c) 2026 Mirrowel
+
+# src/rotator_library/providers/anthropic_provider.py
+
+import copy
+import hashlib
+import json
+import os
+import time
+import logging
+import uuid
+from typing import Union, AsyncGenerator, List, Dict, Any, Optional
```

Ruff (0.15.4)

[warning] Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)